docs/topics/strings.md

```diff
@@ -132,4 +141,48 @@ use the following syntax:
 val price = """
 ${'$'}_9.99
 """
-```
\ No newline at end of file
+```
+
+## String formatting
+
+> String formatting with the `String.format()` function is available for the Kotlin/JVM platform.
+>
+{type="note"}
+
+To format a string to your specific requirements, use the [`String.format()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/format.html) function. 
+
+The `String.format()` function accepts a format string and one or more arguments. The format string contains one placeholder 
+(`%`) for each remaining argument, followed by format specifiers. Format specifiers are formatting instructions for the 
```

Comment:
```suggestion
(`%`) for each remaining argument, followed by [format specifiers](https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html#summary). Format specifiers are formatting instructions for the 
```
Until the API reference is documented, I think we need a link to where format specifiers are defined in Java's documentation.

Reply:
Agree. @ilya-g Please let me know when the API is updated and includes the link to Java's docs, then I remove it from here. Thanks in advance!